Descent3/gamesave.cpp

```diff
@@ -319,23 +319,26 @@ extern int Times_game_restored;
 // available for all.
 int Quicksave_game_slot = -1;
 
+// Quicksave function
```

Comment:
Most added comments in this file are pointless and only add noise

Reply:
Well you should learn to read code. `Game_mode & GM_MULTI` is as clear as `// Check if game is in multiplayer mode`, etc. If code is not understandable easily, it's likely because of bad naming. Read Clean Code chap 2 and 4.